src/plugins/ui_stateful/clipboard.ts

```diff
@@ -46,6 +46,7 @@ export class ClipboardPlugin extends UIPlugin {
   private lastPasteState?: ClipboardState;
   private _isPaintingFormat: boolean = false;
   private originSheetId?: UID;
+  private keepPaintingFormat: boolean = false;
```

Comment:
```suggestion
  private isPaintFormatPersistent: boolean = false;
``` 
What do you think about this name ?